js/index.js

```diff
@@ -142,3 +142,24 @@ function hideShow () {
 //function call to hide or show message list when page loads
 hideShow();
 
+let gitHubRequest = new XMLHttpRequest();
+
+gitHubRequest.open("GET", "https://api.github.com/users/nicoleCodeGirl/repos", true);
+gitHubRequest.send();
+
+gitHubRequest.onload = function() {
+    let repositories = JSON.parse(this.response);
+    console.log(repositories);
+
+  
+let projectSection  = document.getElementById("projects");
+
+let projectList = projectSection.querySelector("ul");
+
+for (let i = 0; i < repositories.length; i++){
+    let project = document.createElement("li");
+    project.innerHTML = `<a href="https://www.github.com/nicolecodegirl/${repositories[i].name}">` + repositories[i].name + '</a>';
```

Comment:
In the API response, there's another property that will give you the full URL: Just another option.